### PR TITLE
Add base support for vec6

### PR DESCRIPTION
**Pull request by thekemkid**
As per issue #578 this library needs support for vec6. This is a PR to add that support.

### CODE REVIEW

File: cc/core/Vec.cc [hunk 1]
```diff
@@ -39,42 +40,74 @@ NAN_MODULE_INIT(Vec::Init) {
...
+		case 6:
+		  jsVec = FF::newInstance(Nan::New(Vec6::constructor));
+		  Nan::ObjectWrap::Unwrap<Vec6>(jsVec)->self = cv::Vec6d(
+		  	info[0]->ToNumber(Nan::GetCurrentContext()).ToLocalChecked()->Value(),
+		  	info[1]->ToNumber(Nan::GetCurrentContext()).ToLocalChecked()->Value(),
+		  	info[2]->ToNumber(Nan::GetCurrentContext()).ToLocalChecked()->Value(),
+		  	info[3]->ToNumber(Nan::GetCurrentContext()).ToLocalChecked()->Value(),
+		  	info[4]->ToNumber(Nan::GetCurrentContext()).ToLocalChecked()->Value(),
+		  	info[5]->ToNumber(Nan::GetCurrentContext()).ToLocalChecked()->Value()
+		  );
```
Comment (thekemkid) [inline cc/core/Vec.cc]: I wanted to rewrite this using macros to minimise the duplication but I'm too unfamiliar in cpp land to do it correctly, so this is the alternative I came up with.

Comment (justadudewhohacks) [reply cc/core/Vec.cc]: No worries, no need to introduce macros here :)

**Comment by justadudewhohacks** [comment]
Great thanks for the PR, looks good to me! Just a few minor things. :)

Could you add the typings for Vec6, similar to [Vec4.d.ts](https://github.com/justadudewhohacks/opencv4nodejs/blob/master/lib/typings/Vec4.d.ts). 

Furthermore, since you added support for Vec6 in the cv.partition binding, could you also add the typing [here ](https://github.com/justadudewhohacks/opencv4nodejs/blob/c97f71fa1d2830dd7ac74d6f2fdd10787cab75aa/lib/typings/cv.d.ts#L141) and the testcase [here](https://github.com/justadudewhohacks/opencv4nodejs/blob/02be3b6416b8225efca4d564d64d5db71372babd/test/tests/core/coreTests.js#L84),

**Comment by thekemkid** [comment]
Nice, feedback is incorporated now! :)

While in there I noticed that Vec4's typing was wrong for the constructor, so I fixed it.

**Comment by justadudewhohacks** [comment]
Great, thanks!

**Pull request merged**